hydra-node/src/Hydra/HeadLogic/State.hs

```diff
@@ -212,6 +218,32 @@ seenSnapshotNumber = \case
   RequestedSnapshot{lastSeen} -> lastSeen
   SeenSnapshot{snapshot = Snapshot{number}} -> number
 
+-- | A deposit tracked by the protocol. The 'DepositStatus' determines whether
+-- it may be used for an incremental commit or not.
+data Deposit tx = Deposit
+  { headId :: HeadId
+  , deposited :: UTxOType tx
+  , deadline :: UTCTime
+  , status :: DepositStatus
+  }
+  deriving (Generic)
+
+deriving stock instance IsTx tx => Eq (Deposit tx)
+deriving stock instance IsTx tx => Show (Deposit tx)
+deriving anyclass instance IsTx tx => ToJSON (Deposit tx)
+deriving anyclass instance IsTx tx => FromJSON (Deposit tx)
+
+instance ArbitraryIsTx tx => Arbitrary (Deposit tx) where
+  arbitrary = genericArbitrary
+  shrink = recursivelyShrink
+
+data DepositStatus = Unknown | Active | Expired
```

Comment:
suggest to rename Unknown for New/Inactive

Reply:
It is called `Inactive` in the next PR :)